portfolio/src/main/java/com/google/sps/servlets/CovidDataServlet.java

```diff
@@ -0,0 +1,64 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/covid-data")
+public class CovidDataServlet extends HttpServlet {
+
+  // Maps: Country -> (Date -> Confirmed)
+  private Map<String, Map<String, Integer>> covidCases = new HashMap<>();
+
+  @Override
+  public void init() {
+    Scanner scanner =
+        new Scanner(getServletContext().getResourceAsStream("/WEB-INF/covid_cases.csv"));
+    while (scanner.hasNextLine()) {
+      String line = scanner.nextLine();
+      String[] cells = line.split(",");
+
+      String date = cells[0];
+      String country = cells[1];
+      Integer confirm = Integer.valueOf(cells[2]);
+
+      if (!covidCases.containsKey(country)) {
+        Map<String, Integer> map = new HashMap<>();
+        covidCases.put(country, map);
+      }
```

Comment:
Instead of this block, you can use `getOrDefault()` below?

Reply:
I changed to `putIfAbsent()` instead